tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/action/WatchWorkspaceModulesAction.java

```diff
@@ -126,6 +130,12 @@ else if ("stop".equals(_action)) {
 					if (bundleProject != null) {
 						try {
 							gogoBundleDeployer.uninstall(bundleProject);
+
+							IFolder folder = FileUtil.getFolder(selectedProject, "build");
```

Comment:
Lets add a couple of "if file exists" checks here just to avoid an error when we try to delete.